guidelines.mdx

```diff
@@ -0,0 +1,518 @@
+
+### Consistency
+
+We aim for the highest possible level of consistency in the documentation when it comes to the application of these guidelines, and particularly:
+- The wording used when referring to Scaleway products and features
+- The structure and formatting of our documentation pages
+- The correct use of grammar and syntax
+
+Inconsistency can result in confusion and misinterpretation, while documentation that applies consistent standards provides a streamlined user experience, and underlines the professionalism and reliability of Scaleway.
+
+## Page types, formats and structure
+
+### Page types
+
+All documentation pages must fall into one of the following categories:
+
+| PAGE TYPE          | DESCRIPTION                                                                              |
+| -------------------| ---------------------------------------------------------------------------------------- |
+| How to             | Shows how to do a core action (e.g. create, configure, delete, use features) for a Scaleway product via the console, e.g. [How to create a Load Balancer](/network/load-balancer/how-to/create-load-balancer/)                                        |
+| Quickstart         | Shows how to do the main series of actions necessary to start using a Scaleway product via the console, e.g. [Instances Quickstart](/compute/instances/quickstart/)                                   |
+| Concepts           | Defines and briefly explains the main terminology associated with a Scaleway product, e.g. [Kubernetes Concepts](/containers/kubernetes/concepts/)                                            |
+| Troubleshooting    | Gives solutions to help resolve a common issue with a Scaleway product, e.g. [VPC auto-config is not working](/network/vpc/troubleshooting/autoconfig-not-working/)                          |
+| API/CLI          | Presents information and tips for creating and configuring a Scaleway product via both official and third-party APIs, CLIs, SDKs and other developer tools, e.g. Terraform, e.g. [Using Object Storage with the AWS-CLI](/storage/object/api-cli/object-storage-aws-cli/). These pages complement the [main developer reference documentation](https://www.scaleway.com/en/developers/).                           |
+| Additional Content | Provides detailed information on specific subjects and features associated with a Scaleway product, e.g. [Optimize your Object Storage performance](/storage/object/reference-content/optimize-object-storage-performance/)                                            |
+| Tutorial           | Explains how to use a Scaleway product with a third-party tool, or how to use multiple Scaleway products together for a specific use-case, e.g. [Setting up a Proxmox cluster on Elastic Metal servers using Private Networks](/tutorials/setup-elastic-metal-proxmox-cluster-with-private-networks/)                                            |
```

Comment:
Maybe put another example here, this tuto is very complex and maybe not the best example

Reply:
```suggestion
| Tutorial           | Explains how to use a Scaleway product with a third-party tool, or how to use multiple Scaleway products together for a specific use-case, e.g. [Creating WordPress Instances with the Scaleway CLI
](/tutorials/create-wordpress-instances-cli/)                                            |
```